.travis.yml

```diff
@@ -4,20 +4,12 @@ language: python
 
 matrix:
   include:
-    - python: 2.7
-      env: TOX_ENV=py27-django-111
-    - python: 3.4
-      env: TOX_ENV=py34-django-111
-    - python: 3.4
-      env: TOX_ENV=py34-django-20
-    - python: 3.5
-      env: TOX_ENV=py35-django-111
-    - python: 3.5
-      env: TOX_ENV=py35-django-20
     - python: 3.6
-      env: TOX_ENV=py36-django-111
-    - python: 3.6
-      env: TOX_ENV=py36-django-20
+      env: TOX_ENV=py36-django22
+    - python: 3.7
+      env: TOX_ENV=py37-django22
+    - python: 3.8
+      env: TOX_ENV=py38-django22
```

Comment:
I guess, this should also include at least some kind of testing for Django 3.0 in CI?

Reply:
Indeed, thanks!